CONTRIBUTING.md

```diff
@@ -1,22 +1,120 @@
-# Contributing to volplugin
+# Contributing to Volplugin
 
 Patches and feature additions are totally welcome to volplugin! For now, please
 see the [README](https://github.com/contiv/volplugin/blob/master/README.md) for
 build and test instructions.
 
-If you have a patch to contribute to volplugin, please follow these guidelines:
+There are other ways to contribute, not just code and feature additions
+- report issues, proposing documentation and design changes, submit bug
+fixes, propose design changes, discuss use cases or become a maintainer.
 
-* Sign your commits with `git commit -s`. Note that we live under an Apache
-  license.
-* Your comments should detail the directory they are affecting, as below. This
-  makes it much simpler than `git log --stat` to determine what areas of a
-  project a specific commit is referencing.
-  * Example: `librbd: changed a widget`
-* Please squash your commits per fix/feature, with a single commit message.
+## Submitting pull requests to change the documentation or the code
```

Comment:
gah, forgot this, sorry. Please leave the restriction in about directory prefixing the commit messages.